tests/datatypes/test_bytes.py

```diff
@@ -20,6 +20,20 @@ def test_getattr(self):
                 print(err)
             """)
 
+    def test_capitalize(self):
+        self.assertCodeExecution("""
+            print(b'hello, world'.capitalize())
+            print(b'helloWORLD'.capitalize())
+            print(b'HELLO WORLD'.capitalize())
+            print(b'2015638687'.capitalize())
```

Comment:
Sounds like a good approach to me. Might also be worth flagging that the expected failure test case can be merged into the general case test once the failure is resolved.